tests/framework/driver/data_driven_agent_test_driver.py

```diff
@@ -27,14 +27,15 @@
 from neuro_san.message_processing.basic_message_processor import BasicMessageProcessor
 from neuro_san.session.direct_agent_session import DirectAgentSession
 
-from tests.neuro_san.client.agent_evaluator import AgentEvaluator
-from tests.neuro_san.client.agent_evaluator_factory import AgentEvaluatorFactory
-from tests.neuro_san.client.assert_forwarder import AssertForwarder
+from tests.framework.evaluators.agent_evaluator_factory import AgentEvaluatorFactory
+from tests.framework.interfaces.agent_evaluator import AgentEvaluator
+from tests.framework.interfaces.assert_forwarder import AssertForwarder
 
 
-class DataDrivenAgentTest:
+class DataDrivenAgentTestDriver:
     """
-    Abstract test class whose subclasses define hocon file to parse as test cases.
+    Class which manages the execution of a single data-driven test case
+    specified as a hocon file.
     """
```

Comment:
Tweak the name and comment of this class to better describe what it is becoming.